docs/_templates/base.html

```diff
@@ -75,6 +75,14 @@
     {%- block extra_styles -%}
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
     <link rel="preconnect" href="https://WLG0MZB58Q-dsn.algolia.net" crossorigin />
+
```

Comment:
I think this should be moved below the `{%- block extrahead -%}{%- endblock -%}` line (currently line 91 in this PR). Currently this is inside the `extra_styles` block.

Reply:
good catch, I'll get this updated.